packages/react/src/components/Table/mdx/Table.mdx

```diff
@@ -470,6 +470,7 @@ import TableViewDropdownContent from '../TableViewDropdown/TableViewDropdownCont
 | locale               | string          |         | what locale should we use to format table values if left empty no locale formatting happens                                                                                                            |
 | i18n                 | object          |         | (see [i18n prop](#i18n-prop))                                                                                                                                                                          |
 | error                | string          |         | Specify the error message that need to be displayed by default. Incase we use `view.table.errorState` property then the error state element will be rendered instead of error message                  |
+| noDataIcon           | string          |         | While Table is empty then we have choice while icon should be display. one of (error', 'error404', 'empty', 'not-authorized', 'no-result', 'success') option so display.                               |
```

Comment:
`When the Table is empty, we have the choice of which icon should be displayed. One of the following options ('error', 'error404', 'empty', 'not-authorized', 'no-result', 'success') or a custom function.`